### PR TITLE
Build preRender DigitalSubscriptionLandingPage and simplify preRender SupporterPlusLandingPage

### PR DESCRIPTION
**Pull request by paul-daniel-dempsey**
## What are you doing in this PR?

CORRECTION REQUESTED 13/1/23 GeorgeH : 

Simplified Process, pre-render only basic page to  encompass all future checkout variants without modifcitsion
- Create (separate) preDigitalSubscriptionLandingPage
- Modify preSupporterPlusLandingPage 
1.  Remove LeftHand side text
2.  Remove Both White Stacked Containers

[**archived un-released PR**](https://github.com/guardian/support-frontend/pull/5512)

## Why are you doing this?

So users on slower devices (eg. Mobile devices not on WiFi) don’t just get a blank screen when they first land on the checkout.

### CODE REVIEW

File: support-frontend/assets/pages/supporter-plus-landing/preRenderSupporterPlus.tsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import type { CountryGroupId } from 'helpers/internationalisation/countryGroup';
+import { GBPCountries } from 'helpers/internationalisation/countryGroup';
+import { PreRenderSupporterPlusLandingPage } from './preRenderSupporterPlusLandingPage';
+
+export const supporterPlusLanding = (countryGroupId?: CountryGroupId) => (
```
[the remark this answers is not in the document]
Comment (paul-daniel-dempsey) [reply support-frontend/assets/pages/supporter-plus-landing/preRenderSupporterPlus.tsx:5]: Was in 2 minds over this, country selector dropdown always presumes UK which is not always the case so this seemed a halfway house. Can see its not used howveer so sticking to that rule I will remove the last commit with all this in.

[the remark this answers is not in the document]
Comment (paul-daniel-dempsey) [reply support-frontend/assets/pages/supporter-plus-landing/preRenderSupporterPlus.tsx]: Removed this

File: support-frontend/assets/pages/supporter-plus-landing/components/landingPageHeading.tsx [hunk 2]
```diff
@@ -1,37 +1,20 @@
...
-		margin-bottom: ${space[3]}px;
+		display: inline-block;
 	}
 `;
 
 type Props = { heading?: string | JSX.Element };
-export function LandingPageHeading({
-	heading = 'Support fearless, independent journalism',
-}: Props): JSX.Element {
+export function LandingPageHeading({ heading = '' }: Props): JSX.Element {
```
[the remark this answers is not in the document]
Comment (paul-daniel-dempsey) [reply support-frontend/assets/pages/supporter-plus-landing/components/landingPageHeading.tsx]: `support-frontend/assets/pages/supporter-plus-landing/preRenderSupporterPlusLandingPage.tsx` now does not import `LandingPageHeading` as mentioned above, its simply not used (reduce unnecessary markup). `LandingPageHeading` now requires a string to display (ie its not optional).

**Review by jamesgorrie — APPROVED** [review]
Looks great!

**Pull request merged**